src/libxtp/qmpackages/orca.cc

```diff
@@ -825,12 +825,38 @@ std::string Orca::WriteMethod() const {
   std::string convergence =
       this->_convergence_map.at(_settings.get("convergence_tightness")) +
       "SCF ";
-  stream << "! DFT " << _settings.get("functional") << " " << convergence
+  stream << "! DFT " << this->GetOrcaFunctionalName() << " " << convergence
          << opt
          // additional properties provided by the user
          << _settings.get("orca.method") << "\n";
   return stream.str();
 }
 
+std::string Orca::GetOrcaFunctionalName() const {
+
+  char* votca_share = getenv("VOTCASHARE");
+  if (votca_share == nullptr) {
+    return _settings.get("functional");
```

Comment:
I do not understand the logic here.

Reply:
The `VOTCASHARE` environmental variable is not defined for testing, so we need to take into account that  corner case

Reply:
yeah environment variables and testing are a shit combination. Maybe we have to think about a general solution for that. Any ideas?